timeboost-sequencer/src/decrypt.rs

```diff
@@ -505,17 +566,20 @@ impl Worker {
         let Some(per_ct_opt_dec_shares) = self.dec_shares.get(&round) else {
             return Ok(None);
         };
+        let Some(committee) = self.round_committee(round) else {
```

Comment:
```suggestion
        let Some(committee) = self.round_committee(round).cloned() else {
```
With this you do not need the explicit clone below.

Reply:
thanks.
fixed in [abb68a0](https://github.com/EspressoSystems/timeboost/pull/400/commits/abb68a07ebdd6fbc1d2ffde09a7a51bc137fac1a)